grimshot

```diff
@@ -153,6 +163,10 @@ else
   die "Unknown subject to take a screen shot from" "$SUBJECT"
 fi
 
+if [ "$WAIT" != "no" ]; then
```

Comment:
What word could be used for it instead of "no"?
I think `if ( something != "no" )` is not that good.

Reply:
I just used no because the other variable was also initialised with no, it could be basically anything

Reply:
Okay I guess it would be okay